code/datums/skills/mining.dm

```diff
@@ -5,7 +5,7 @@
 /datum/skill/mining/get_skill_speed_modifier(level)
 	switch(level)
 		if(SKILL_LEVEL_NONE)
-			return 1.3
```

Comment:
wtf is this magic number in the first place

Reply:
That's now how modifiers work noob